### DISCUSSION TITLE
Create universal routes to be accessed even without tenant.

### DISCUSSION THREAD

**Discussion by iabduul7** [spatie/laravel-multitenancy#376] [Q&A]
Is there any way to create a `universal` i.e. `global` routes that can be accessed even if I have not set any tenant. For example `localhost:8000/xyz/abc` is without any tenant

**Comment by masterix21** [comment]
Do you need landlord’s routes?

**Reply by iabduul7** [reply]
One way to put it, Yes, the landlord routes

**Reply by masterix21** [reply]
```php
Route::domain('your-landlord-domain.test')->group(function () { 
    //...
});
```
